modules/accounts/email.nix

```diff
@@ -207,6 +207,12 @@ let
         description = "The email address of this account.";
       };
 
+      aliases = mkOption {
+        type = types.listOf (types.strMatching ".*@.*");
+        example = [ "webmaster@example.org" "admin@example.org" ];
+        description = "Alternative email addresses of this account.";
+      };
```

Comment:
you should add a default

Reply:
Yes, like @teto pointed out there should be a `default = []` here. Otherwise there will be issues for people not using aliases.